hummingbot/strategy/liquidity_mining/liquidity_mining.py

```diff
@@ -275,6 +280,24 @@ def start(self, clock: Clock, timestamp: float):
     def stop(self, clock: Clock):
         pass
 
+    def _validate_order_book_for_markets(self):
+        """
+        Verify that the active markets have non-empty order books. Put the trading on hold
+        for pairs with empty order books, in case it is a glitch?
+        """
+        markets = self._market_infos.copy()
+        markets.update(self._empty_ob_market_infos)
+        for market, market_info in markets.items():
+            if markets[market].get_price(False).is_nan():
+                self._empty_ob_market_infos[market] = market_info
+                if market in self._market_infos:
+                    self._market_infos.pop(market)
```

Comment:
Just one simple change, we should log that this market is going to be removed or which markets are unable

Reply:
@cardosofede Good point, I log it as a warning (rather than info) since the user may want to stop and redefine the confi rather than rely on the bot mechanic